test/unit/chat/signature.test.ts

```diff
@@ -0,0 +1,21 @@
+import { CER, SEC, getMsgHash, serializeMsg, signMsg, verifySignature } from '../../../src/chat/signature'
+import { sha256 } from '@noble/hashes/sha256'
+
+const PRIV_KEY = 'a11bc5d2eee6cdb3b37f5473a712cad905ccfb13fb2ccdbf1be0a1ac4fdc7d2a'
+const PUB_KEY = '023a9da707bee1302f66083c9d95673ff969b41607a66f52686fa774d64ceb87'
+
+describe('signature', () => {
+  it('serializeMsg', () => {
+    const msg = { id: 'test' }
+    const res = serializeMsg(msg)
+    expect(res).toEqual(JSON.stringify(msg))
+  })
+  it.skip('sign and verifySignature', () => {

```

Comment:
@angelo-v I'm having trouble with this test.  I've tried several different things. I was wondering if you can tell just by looking what I'm doing wrong. Here is the error.
`[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "undefined".] {
  code: 'ERR_UNHANDLED_REJECTION'`

Reply:
I do not have your issue but `TypeError: Expected input type is Uint8Array (got object)`
This seems ajest/jsdom issue with asolution proposed in https://github.com/paralleldrive/cuid2/issues/44#issuecomment-1531731695

Reply:
Thanks @bourgeoa I will check that out.